tests/Aspire.Npgsql.EntityFrameworkCore.PostgreSQL.Tests/ConformanceTests_NoPooling.cs

```diff
@@ -13,11 +15,12 @@ public class ConformanceTests_NoPooling : ConformanceTests_Pooling
 
     protected override void RegisterComponent(HostApplicationBuilder builder, Action<NpgsqlEntityFrameworkCorePostgreSQLSettings>? configure = null, string? key = null)
     {
-        builder.AddNpgsqlDbContext<TestDbContext>("postgres", settings =>
+        // Configure Npgsql.EntityFrameworkCore.PostgreSQL services
+        if (builder.Configuration.GetConnectionString("postgres") is string { } connectionString)
         {
-            settings.DbContextPooling = false;
+            builder.Services.AddDbContextPool<TestDbContext>(dbContextOptionsBuilder => dbContextOptionsBuilder.UseNpgsql(connectionString));
```

Comment:
Do we still need Pooling vs NoPooling tests now that we don't have the concept in the component anymore?

Reply:
I am merging them.